test/less/declaration/2.json

```diff
@@ -0,0 +1,81 @@
+{
+  "type": "declaration",
+  "content": [
+    {
+      "type": "property",
+      "content": [
+        {
+          "type": "ident",
+          "content": "@{prop}",
```

Comment:
`ident` is something different. Here is grammar from the spec:
http://www.w3.org/TR/css3-selectors/#lex

And according to Less's spec, `@{prop}` is an `interpolatedVariable`:
http://lesscss.org/features/#variables-feature-variable-interpolation
https://github.com/tonyganch/gonzales-pe/blob/dev/docs/node-types.md#interpolatedvariable